### DISCUSSION TITLE
Hi, may i ask for a permission to push my branch with message ru localization for incorrect UUID

### DISCUSSION THREAD

**Discussion by ice1x** [encode/django-rest-framework#7884] [General]
## Checklist

I am trying to send localized message for here: https://github.com/encode/django-rest-framework/blob/master/rest_framework/locale/ru/LC_MESSAGES/django.po#L217

$ git push --set-upstream origin localize-invalid-uuid-ru-message
remote: Permission to encode/django-rest-framework.git denied to ice1x.
fatal: unable to access 'https://github.com/encode/django-rest-framework.git/': The requested URL returned error: 403

**Comment by lovelydinosaur** [comment]
For translations you need to [use transifex](https://www.transifex.com/django-rest-framework-1/django-rest-framework/)

(Also, for code changes use GitHub's pull requests, rather than pushing directly.)

**Reply by ice1x** [reply]
Hi, thank you for the recomendations!
I have already pushed a pull request:
https://github.com/encode/django-rest-framework/pull/7885

Also tried transifex but there is no russian in dropdown during "join team" processing but it is exists in a list of project languages:
